job-server/src/main/scala/spark/jobserver/WebApi.scala

```diff
@@ -345,7 +340,9 @@ class WebApi(system: ActorSystem,
       get { ctx =>
         (supervisor ? ListContexts).mapTo[Seq[String]]
           .map { contexts =>
-            ctx.complete(removeProxyUserPrefix(authInfo, contexts)) }
+            ctx.complete(SparkJobUtils.removeProxyUserPrefix(
+              authInfo.toString, contexts,
```

Comment:
What is `authInfo.toString` if auth is not used, which is the majority use case probably?